lib/facter/selinux_semanage_is_python3.rb

```diff
@@ -1,6 +1,6 @@
 Facter.add(:selinux_semanage_is_python3) do
   confine osfamily: 'RedHat'
   setcode do
-    Facter::Core::Execution.execute('rpm -q libsemanage-python3') !~ %r{not installed}
+    Facter::Core::Execution.execute('rpm -q python3-libsemanage') !~ %r{not installed}
```

Comment:
@bastelfreak Note this is *specific* to python3 version of the package, not the python 2 version of the package. CentOS 6 and 7 should play no role in using this fact.